### PR TITLE
Add tolerance for more decimal places in style tags

### PR DESCRIPTION
**Pull request by anurag90x**

### CODE REVIEW

File: tests/documents/paragraphs_with_images.html [hunk 1]
```diff
@@ -9,9 +9,9 @@
...
   <img class="fr-fir fr-dii fr-draggable" width="500" src="/service/uploads/group/1338761/bundles/0213163b-6e70-4f41-8809-39392f7a7474/files/10bc24d1-634e-46bb-8fa5-eb7030e1b40b/240/170/fit" alt="Editor photo" style="display: inline">
 
-  <img class="fr-fir fr-dii fr-draggable" width="500" src="/service/uploads/group/1338761/bundles/0213163b-6e70-4f41-8809-39392f7a7474/files/10bc24d1-634e-46bb-8fa5-eb7030e1b40b/240/170/fit" alt="Editor photo" style="width: 200.123px;height: 345.15px;display: block;">
+  <img class="fr-fir fr-dii fr-draggable" width="500" src="/service/uploads/group/1338761/bundles/0213163b-6e70-4f41-8809-39392f7a7474/files/10bc24d1-634e-46bb-8fa5-eb7030e1b40b/240/170/fit" alt="Editor photo" style="width: 200.123px;height: 345.159999px;display: block;">
 </p>
 
 <p><img src="/service/uploads/images/test" style="width: 300px;;"/></p><p><br/><img src="/service/uploads/images/test" style="width: 300px;;"/></p>
 
-<p><img src=\"/service/uploads/images/war\" style=\"width: 188px; height: 132.85px;;\"/>&#160;qweqeqeqw</p>
+<p><img src=\"/service/uploads/images/war\" style=\"width: 188.999991px; height: 132.8511111px;;\"/>&#160;qweqeqeqw</p>
```
Comment (markstory) [inline tests/documents/paragraphs_with_images.html]: So close to 189 :smile:

File: markup_language/input_transformer.py [hunk 2]
```diff
@@ -11,10 +11,7 @@ def load_xslt():
 
 
 def wrap_in_container(xml_string):
-    if not xml_string.startswith("<div") and \
-       not xml_string.endswith("</div>"):
-            xml_string = "<div>" + xml_string + "</div>"
-    return xml_string
+    return "<container>" + xml_string + "</container>"
```
Comment (markstory) [inline markup_language/input_transformer.py]: Should we use an HTML element that exists? Perhaps `<section>`?

Comment (anurag90x) [inline markup_language/input_transformer.py]: I wanted to make one up to explicitly avoid conflict with the actual document content and then strip it out before returning.

Comment (markstory) [inline markup_language/input_transformer.py]: ok.

**Pull request merged**